index.html

```diff
@@ -13,16 +13,17 @@
 <table class="table table-bordered table-condensed table-hover table-striped">
 <thead><tr>
 <th></th>
-<th><a href="https://github.com/jakubpawlowicz/clean-css.git">clean-css - 3.2.0</a></th>
-<th><a href="https://github.com/jakubpawlowicz/clean-css.git">clean-css (advanced off) - 3.2.0</a></th>
+<th><a href="git+https://github.com/jakubpawlowicz/clean-css.git">clean-css - 3.2.5</a></th>
```

Comment:
> `git+https`?

I didn't edit `index.html`. It's just generated with `npm run bench` command. I removed `git+https` by hand though.

```
$ node -v
$ npm run bench

> css-minification-benchmark@1.0.0 bench /Users/Shinnosuke/github/css-minification-benchmark
> node ./bin/bench --html>index.html

....................B................................F.........................................F....................................................F...................
```